src/coreclr/vm/threadsuspend.cpp

```diff
@@ -5873,6 +5873,13 @@ void Thread::ApcActivationCallback(ULONG_PTR Parameter)
     ActivationReason reason = (ActivationReason)pData->Parameter;
     PCONTEXT pContext = pData->ContextRecord;
 
+#if defined(TARGET_ARM64)
+    // Windows incorrectly set the CONTEXT_UNWOUND_TO_CALL in the flags of the context it passes to us.
```

Comment:
Should we report it as a bug to Windows?

Reply:
Yes, I am planning to do that.